doc/acl/Everflow-test-plan.md

```diff
@@ -0,0 +1,595 @@
+
+#### Test objective
+
+Verify that session with resolved ECMP route has active state.
+
+#### Test steps
+
+- Create ECMP route that matches session destination IP with two unresolved next hops.
+- Resolve route next hops.
+- Verify that session state in APP DB is changed to active.
+- Send packets that matches each Everflow ACL rule.
+- Verify that packets mirrored to appropriate port.
+- Analyze mirrored packets header.
+- Verify that mirrored packets payload is equal to sent packet.
+
+### Test case \#6 - ECMP route change (add next hop).
+
+#### Test objective
+
+Verify that insertion of additional next hop to ECMP group doesn't affects session DST MAC and port.
+
+#### Test steps
+
+- Create ECMP route that matches session destination IP with two unresolved next hops.
+- Resolve route next hops.
+- Verify that session state in APP DB is changed to active.
+- Send packets that matches each Everflow ACL rule.
+- Verify that packets mirrored to appropriate port.
+- Analyze mirrored packet header.
+- Verify that mirrored packet payload is equal to sent packet.
+- Add resolved next hop to ECMP route.
+- Send packets that matches each Everflow rule.
+- Verify that packets are mirrored to the same port.
+- Verify that mirrored packets have the same DST MAC.
+
+### Test case \#7 - ECMP route change (remove next hop used by session).
+
+#### Test objective
+
+Verify that removal of next hop that is not used by session doesn't cause DST port and MAC change.
+
+#### Test steps
+
+- Create ECMP route that matches session destination IP with two unresolved next hops.
+- Resolve route next hops.
+- Verify that session state in APP DB is changed to active.
+- Send packets that matches each Everflow rule.
+- Verify that packets mirrored to appropriate port.
+- Analyze mirrored packet header.
+- Verify that mirrored packets payload is equal to sent packets.
+- Remove next hop that is not used by session.
+- Send packets that matches each Everflow rule.
+- Verify that packets are mirrored to the same port.
+- Verify that mirrored packets have the same DST MAC.
+
+### Test case \#8 - ECMP route change (remove next hop not used by session).
+
+#### Test objective
+
+Verify that after removal of next hop that was used by session from ECMP route session state is active.
+
+#### Test steps
+
+- Create ECMP route that matches session destination IP with two unresolved next hops.
+- Resolve route next hops.
+- Verify that session state in APP DB is changed to active.
+- Send packets that matches each Everflow ACL rule.
+- Verify that packets mirrored to appropriate port.
+- Analyze mirrored packet header.
+- Verify that mirrored packets payload is equal to sent packets.
+- Remove next hop that is used by session.
+- Send packets that matches each Everflow ACL rule.
+- Verify that packets are mirrored and destination port changed accordingly.
+
+### Other possible tests
+
+## TODO
+- Everflow+LAG test configuration and testcases (separate ansible playbook)
```

Comment:
Please add Everflow+VLAN test configuration and testcases. Mainly this is about destination port movement within/out VLAN scope.